packages/account/src/Components/leave-confirm/leave-confirm.tsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import { RouteComponentProps, useHistory, withRouter } from 'react-router-dom';
```

Comment:
Can you please Ignore this file as I am also working on this. 👀 

CC: @amina-deriv

Reply:
i have removed my changes @fasihali-deriv 👍

Reply:
Thank youo @Dhruv-deriv 🙏